doc/user-manual/tools/compilers.lagda.rst

```diff
@@ -157,6 +151,10 @@ Options
      The files will be placed into the root directory of the compiled Agda project,
      or into the directory given by :option:`--compile-dir`.
 
+.. option:: --js-es6
+
+    Produce ES6 style modules.
+
```

Comment:
This feature is going into Agda 2.8.0.  Please add "since 2.8.0" to this option description (see option descriptions in `command-line.rst`).

Reply:
Done in c40ccd4

Reply:
Done properly in 9ab3e16